src/main/java/at/hannibal2/skyhanni/utils/renderables/Renderable.kt

```diff
@@ -477,30 +483,22 @@ interface Renderable {
             }
         }
 
-        fun singeltonString(string: String): List<Renderable> {
-            return Collections.singletonList(string(string))
-        }
-
+        @Deprecated("Use RenderableString instead", ReplaceWith("RenderableString()"))
```

Comment:
add the parameters to the ReplaceWith function call so that the intellij 
action "replace with" works correctly, or remove replace with entirely.